persistence-worker/main.py

```
@@ -0,0 +1,134 @@
+                        r["price"],
+                        r["volume"],
+                    )
+                    for r in current_batch
+                ]
+                await conn.copy_records_to_table(
+                    "stock_ticks",
+                    columns=("time", "symbol", "stock_code", "price", "volume"),
+                    records=records,
+                )
+            print(f"Saved {len(current_batch)} rows to database")
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)